languages/csharp/exercises/concept/constants/.docs/instructions.md

```diff
@@ -0,0 +1,21 @@
+The authentication system that you last saw in (TODO cross-ref-tba) is in need of some attention. You have been tasked with cleaning up the code.
```

Comment:
Maybe you could mention that you'll not only be cleaning up the code, you'll also be hardening it to improve safety or something like that.

Reply:
Done